assets/src/edit-story/components/dropzone/index.js

```diff
@@ -119,6 +120,8 @@ function DropZone({ children, onDrop, pageIndex, dragIndicatorOffset }) {
   const highlightWidth = 5;
   return (
     <DropZoneComponent
+      as={as}
+      role={as === 'li' ? 'option' : null}
```

Comment:
Wouldn't it perhaps be nicer to do this with just `{...rest}` and let the outer component retain control of this?

Reply:
I'm happy to use `{...rest}` here.
I personally use this technique a lot, but not everybody likes it (some cons there).

For this small case, I did the change with the smallest possible impact, if you ever use this element as `li` you probably also want `role="option"`.

Reply:
My rule of thumb: 
if you can somehow derive prop from another prop - do it
pass a single prop if it's just one (maybe convert later to rest props)
pass rest props for multi-props case

Reply:
The `as` props in this file and `draggablePage` all seem now to be unused. I'd therefore revert the changes again.

Reply:
`as` is something native to `styled`, so it is in use.

Reply:
@barklund Yes and no. Here, `as` was used to pass it down to children. See how 565e9e193a26969c53323efc8f252a09ec520d53 changed usage so that `as` is no longer used. Instead, `DraggablePage` (or `Item`) is now wrapped in an `Li`.

Reply:
Remove those lines and in `DraggablePage` and `DropZone` and see that everything still works as expected.

Reply:
I will update the first part of this PR and remove unused stuff since now it's more beneficial to wrap the `page|indicator` in `li` (for easier margin management).